### PR TITLE
Unit testing handler 

### PR DESCRIPTION
**Pull request by SumedhaJoshSoft**

### CODE REVIEW

File: handler_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	res, err := http.DefaultClient.Do(req)
+	assert.NoError(t, err)
+	assert.Equal(t, res.StatusCode, http.StatusOK)
+
+}
+
+/*
+Expecting valid json response having all loaded websites as key and their respetive statuses as value
+*/
+func TestCheckAllSiteStatusHandler(t *testing.T) {
```
Comment (pari-27) [inline handler_test.go]: please give the scenario as the testcase name

File: handler_test.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	err = json.NewDecoder(res.Body).Decode(&websites)
+	assert.Equal(t, err, nil)
+}
+
+/*
+Expecting valid json response with value as "DOWN" for website "cdbcb"
+*/
+func TestCheckSiteStatusHandler(t *testing.T) {
+
+	req, _ := http.NewRequest(http.MethodGet, "http://localhost:8000/websites?name=cdbcb", nil)
```
Comment (pari-27) [inline handler_test.go]: can we mock this rather than using local endpoint

File: handler_test.go [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+	err = json.NewDecoder(res.Body).Decode(&websites)
+	assert.Equal(t, response, websites)
+	assert.Equal(t, err, nil)
+}
+
+/*
+Request body should be a valid json not a empty string or nil
+*/
+func TestLoadWebsitesHandler(t *testing.T) {
+	req, _ := http.NewRequest(http.MethodPost, "http://localhost:8000/websites?fff=cdbcb", nil)
```
Comment (pari-27) [inline handler_test.go]: here also

File: handler_test.go (hunk 1 above)
Comment (SumedhaJoshSoft) [reply handler_test.go]: Noted

File: handler_test.go (hunk 2 above)
Comment (SumedhaJoshSoft) [reply handler_test.go]: Will update and revert

File: handler_test.go (hunk 3 above)
Comment (SumedhaJoshSoft) [reply handler_test.go]: Will update and revert

**Review by pari-27 — APPROVED** [review]

**Pull request merged**